test/read.js

```diff
@@ -92,7 +93,7 @@ tape('read pixels', function (t) {
   })
 
   // now do it for an float fbo
-  if (regl.hasExtension('oes_texture_float')) {
+  if (regl.hasExtension('oes_texture_float') && !safari) {
```

Comment:
Blindly disable safari. Is there any sense to double-check if reading floats is supported here instead of this?

Reply:
Seems okay to me. Could feature-detect it independently from regl, but I think the main goal is just to have safari reasonably report pass/fail on the tests to that regl as a whole is easier to maintain. This accomplishes it, though it'd be nice if there were a `pending` state or something so that it'd report without failing. But short of that, 👍, I guess?

Reply:
I tend to agree on that point. What about exposing ‘regl.supports’ set to let user manually check supported features rather than throwing an error?

Reply:
Seems reasonable if regl detects it anyway. I think `supports` might be nice if we feature-detect to correctly surface errors anyway. `limits `is that it's strictly builtin `getParameter` results (right?) so seems nice to keep that clean.

Side note: Does safari not throw an exception for unsupported types? 😞 See: https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/readPixels#Exceptions

Reply:
Saw the commit. `limits` seems fine too 👍

Reply:
Perfect. Getting a bit lost in the number of issues being tackled here. Got it. 👍 👍